src/openms/source/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderIdentificationAlgorithm.cpp

```diff
@@ -554,6 +552,19 @@ namespace OpenMS
       feature.setMetaValue("label", target_id);
 
       // TODO: Why is the dummy data set here?
```

Comment:
@hendrikweisser Can you comment on the necessity of the dummy ID here?

Reply:
Good point. I think that should be removed (left-over from the previous version that used `IdentificationData` internally, but converted to `PeptideIdentification` for output).